scripts/ci-test.sh

```diff
@@ -29,3 +29,11 @@ source ./scripts/fetch_ext_bins.sh
 fetch_tools
 setup_envs
 make generate test
+# TODO(ncdc): create a separate verify prow job
+make verify
```

Comment:
We have a verify prow job that runs `make verify`, so this shouldn't be needed

Reply:
We actually don't. We have pr-verify, but not a postsubmit or periodic version.

Reply:
Not sure if I missed it, but the comment didn't mention postsubmit/periodic

Reply:
I can reword

Reply:
Actually, I think you're right. CAPA doesn't have a postsubmit verify either. I'll remove it.